### PR TITLE
Ps jira distribution fields

### PR DESCRIPTION
**Pull request by psantos1113**
Add the distribution fields for the Jira Channel. 
Fix some permission issues.

### CODE REVIEW

File: src/main/resources/db/changelog/alert/5.0.0/changelog-configuration-initialization.xml [hunk 1]
```diff
@@ -22,4 +22,16 @@
             CALL DEFINE_FIELD('jira.access.token', TRUE, 'channel_jira', 'GLOBAL');
         </sql>
     </changeSet>
+    <!-- Register Distribution Jira Cloud Fields. -->
+    <changeSet author="psantos" id="1558524516692-3">
+        <sql dbms="h2" stripComments="true">
+            CALL DEFINE_FIELD('jira.add.comments', FALSE, 'channel_jira', 'DISTRIBUTION');
```
Comment (psantos1113) [inline src/main/resources/db/changelog/alert/5.0.0/changelog-configuration-initialization.xml]: For these distribution specific fields do we want channel. as the prefix?  i.e. channel.jira.add.comments
For the HipChat and Slack distribution fields we do that.  So to be consistent we can do that here as well. We also have the FIELD_CONTEXTS table that creates the mapping between the field ID and the Context.

Comment (gkillough) [reply src/main/resources/db/changelog/alert/5.0.0/changelog-configuration-initialization.xml]: I think we should add the prefix for consistency.

Comment (bamandel) [reply src/main/resources/db/changelog/alert/5.0.0/changelog-configuration-initialization.xml]: I would like this as we could have a Jira provider in the future and the prefix would help avoid conflicts.

Comment (psantos1113) [reply src/main/resources/db/changelog/alert/5.0.0/changelog-configuration-initialization.xml]: ok no problem.  I prefer the consistency as well but wanted to make sure I wasn't being too particular.

File: src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDescriptor.java [hunk 2]
```diff
@@ -30,9 +30,13 @@
...
-    public static final String KEY_JIRA_URL = "jira.url";
-    public static final String KEY_JIRA_USERNAME = "jira.user.name";
-    public static final String KEY_JIRA_ACCESS_TOKEN = "jira.access.token";
+    public static final String KEY_JIRA_URL = "jira.cloud.url";
+    public static final String KEY_JIRA_USERNAME = "jira.cloud.user.name";
+    public static final String KEY_JIRA_ACCESS_TOKEN = "jira.cloud.access.token";
+
+    public static final String KEY_ADD_COMMENTS = "channel.cloud.jira.add.comments";
+    public static final String KEY_ISSUE_CREATOR = "channel.cloud.jira.issue.creator";
+    public static final String KEY_JIRA_PROJECT_NAME = "channel.cloud.jira.project.name";
```
Comment (jamesrichard91) [inline src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDescriptor.java]: Why are these field cloud.jira when the other fields have jira.cloud?

File: src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDistributionUIConfig.java [hunk 3]
```diff
@@ -30,18 +30,32 @@
...
 
 @Component
 public class JiraDistributionUIConfig extends ChannelDistributionUIConfig {
+
+    public static final String LABEL_ADD_COMMENTS = "Add Comments";
+    public static final String LABEL_ISSUE_CREATOR = "Issue Creator";
+    public static final String LABEL_JIRA_PROJECT = "Jira Project";
+
+    public static final String DESCRIPTION_ADD_COMMENTS = "If true, this will add comments to the Jira ticket with data describing the latest change to the ticket due to the job processing.";
+    public static final String DESCRIPTION_ISSUE_CREATOR = "The user name of the Jira Cloud user to assign to the issue creator field of the Jira ticket.";
```
Comment (jamesrichard91) [inline src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDistributionUIConfig.java]: The user name of the Jira Cloud user to assign _as_ the issue creator field of the Jira ticket.

File: src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDistributionUIConfig.java [hunk 4]
```diff
@@ -30,18 +30,32 @@
...
 import com.synopsys.integration.alert.common.descriptor.config.ui.ChannelDistributionUIConfig;
 
 @Component
 public class JiraDistributionUIConfig extends ChannelDistributionUIConfig {
+
+    public static final String LABEL_ADD_COMMENTS = "Add Comments";
+    public static final String LABEL_ISSUE_CREATOR = "Issue Creator";
+    public static final String LABEL_JIRA_PROJECT = "Jira Project";
+
+    public static final String DESCRIPTION_ADD_COMMENTS = "If true, this will add comments to the Jira ticket with data describing the latest change to the ticket due to the job processing.";
```
Comment (jamesrichard91) [inline src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDistributionUIConfig.java]: Can we change the end of this to
...with data describing the latest change.

File: src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDescriptor.java (hunk 2 above)
Comment (psantos1113) [reply src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDescriptor.java]: It will be fixed

**Pull request merged**